node/network/availability-recovery/src/tests.rs

```diff
@@ -139,31 +141,48 @@ async fn overseer_recv(
 use sp_keyring::Sr25519Keyring;
 
 #[derive(Debug, Clone)]
-enum HasAvailableData {
+enum Has {
```

Comment:
Should clarify the intended use. `Has` is a tad too ambiguous.

Reply:
it's localized to the test code where the use should be clear